Foundation/NSData.swift

```diff
@@ -930,7 +930,11 @@ open class NSMutableData : NSData {
     }
     
     open func resetBytes(in range: NSRange) {
+        #if os(Android)
+        memset(mutableBytes.advanced(by: range.location), 0, range.length)
+        #else
         bzero(mutableBytes.advanced(by: range.location), range.length)
+        #endif
```

Comment:
I’ll have another look at it. There are some tricky type conversions to get right.